src/main/python/systemml/mlcontext.py

```diff
@@ -34,6 +34,21 @@
 from .converters import *
 from .classloader import *
 
+def getSparkContext():
+    """
+    Internal method to get already initialized SparkContext.
```

Comment:
If this is internal, can you prepend an underscore to the name, i.e., `_getSparkContext`?  Also, `_get_spark_context` would be even better.